### PR TITLE
Add release notes for BUcash 1.5.1.0

### PR DESCRIPTION
**Pull request by sickpig**

### CODE REVIEW

File: doc/release-notes-bucash.md [hunk 1]
```diff
@@ -38,48 +36,158 @@ Other than that upgrading from a version lower than 1.5.0.1 your client is proba
...
-- awemany
+- Andrew Stone
+- Awemany
+- George Bissias
+- Chris Pacia
+- Dimitris Apostolou
 - Greg Griffith
-- Jonathan Toomim
+- Jonas Schnelli
 - Peter Tschipper
```
Comment (ptschip) [inline doc/release-notes-bucash.md:182]: jonas is also an indirect contrib,

File: doc/release-notes/release-notes-bucash1.5.1.0.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+- Andrea Suisani
+- Andrew Stone
+- Awemany
+- George Bissias
+- Chris Pacia
+- Dimitris Apostolou
+- Greg Griffith
+- Jonas Schnelli
+- Peter Tschipper
+- Søren Bredlund Caspersen
```
Comment (gandrewstone) [inline doc/release-notes/release-notes-bucash1.5.1.0.md:183]: jtoomim should be in the list for his path that reduces the preferred timer... even though your configurable patch overrides it.

File: doc/release-notes-bucash.md (hunk 1 above)
Comment (sickpig) [reply doc/release-notes-bucash.md:182]: Agreed

File: doc/release-notes/release-notes-bucash1.5.1.0.md (hunk 2 above)
Comment (sickpig) [reply doc/release-notes/release-notes-bucash1.5.1.0.md:183]: Agreed

**Review by ptschip — APPROVED** [review]

**Pull request merged**